src/view/components/common/login-as/login-as.ts

```diff
@@ -39,7 +36,7 @@ export class LoginAsComponent extends BaseComponent {
     dolLoginAs(options: any): void {
         this.loginService.loginAs(options).subscribe(data => {
             if (data) {
-                this.router.go('/dotCMS');
+                this.router.navigate(['dotCMS']);
```

Comment:
I don't see the value honestly, are you want to do it for all the routes or only "key" routes like login, main, etc..